x-pack/heartbeat/monitors/browser/synthexec/synthtypes.go

```diff
@@ -112,17 +112,36 @@ func (se *SynthError) toMap() common.MapStr {
 	}
 }
 
+type DurationUs struct {
+	Micros int64 `json:"us"`
+}
+
+func (tu *DurationUs) duration() time.Duration {
+	return time.Duration(tu.Micros)
```

Comment:
Had to remove `time.Microsecond` here as Synthetics already spits out them as in microseconds.